lib/tasks/add_navigation_links.rake

```diff
@@ -40,6 +41,7 @@ namespace :navigation_links do
       # [@jeremyf] I went ahead and atomized these tasks so we _could_ call them individually if
       #            desired.  I did not add descriptions so those tasks will not show up in the task
       #            list.
+      Rake::Task["navigation_links:find_or_create:home"].invoke
```

Comment:
We have a bit of a data quandary now.  We are removing the hard-coded link in the main nav, however, for existing Forem's it's unlikely we'll add the replacement link because of the guard condition just above.

Reply:
I guess with this update we will need to add the Home link to existing Forems' DB. We're getting a bit out of the realm of my expertise here though 😅

Reply:
Sounds like we'll need to add a data update script for existing Forems to add it then, so we don't have to rely on this task. Let me know if you need help with this @aitchiss 😃

Reply:
Thanks! I'm certain I'll need help with this 😅  Let me take an initial look at it and then I'll tag you in for assistance 👍